src/main/java/frc/robot/subsystems/SubsystemManager.java

```diff
@@ -8,22 +8,44 @@
 
 public class SubsystemManager {
```

Comment:
we should re-write to store the subsystems in a Map. that way, when we want to do something for all of them, we can loop over the map and save coding bugs where one gets left out.